cmd/server/stateful.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/deepsourcelabs/hermes/config"
+	handler "github.com/deepsourcelabs/hermes/interfaces/http"
+	"github.com/deepsourcelabs/hermes/service"
+	sqlStore "github.com/deepsourcelabs/hermes/storage/sql"
+	"github.com/labstack/echo/v4"
+	log "github.com/sirupsen/logrus"
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+)
+
+func StartStatefulMode(cfg *config.AppConfig, e *echo.Echo) error {
+	db, err := gorm.Open(
+		postgres.Open("postgres://hermes:password@localhost:5432/hermes"),
```

Comment:
👁️

Reply:
The Stateful mode is still a work in progress.  Will move the connection string to the config post that.

Reply:
Will that happen in this PR, or in a subsequent one? If in a subsequent one, can you remove the stateful mode code from this PR?